modules/nuclide-commons-ui/Atomicon.js

```diff
@@ -35,16 +36,21 @@ const TYPE_TO_CLASSNAME_SUFFIX = {
   variable: 'type-variable',
 };
 
-type AtomiconName = $Keys<typeof TYPE_TO_CLASSNAME_SUFFIX>;
+const ICON_NAME_TO_TYPE = invert(TYPE_TO_ICON_NAME);
+
+type AtomiconName = $Keys<typeof TYPE_TO_ICON_NAME>;
```

Comment:
Isn't this `AtomiconType`, not `AtomiconName`?

Reply:
yeah that's a much better name for that type 🤭